### PR TITLE
Add Helper 'index': get elem of array with the index

### PR DESCRIPTION
**Pull request by tkerambloch**

### CODE REVIEW

File: helpers/helpers.go [hunk 1]
```diff
@@ -90,6 +91,16 @@ var ProtoHelpersFuncMap = template.FuncMap{
 	"trimstr": func(cutset, s string) string {
 		return strings.Trim(s, cutset)
 	},
+	"index": func(array interface{}, i int) interface{} {
+		slice := reflect.ValueOf(array)
+		if slice.Kind() != reflect.Slice {
+			panic("index() given a non-slice type")
+		}
+		if slice.Len() > i && i >= 0 {
```
Comment (abronan) [inline helpers/helpers.go]: I would change this portion to:
```go
if i < 0 || i >= slice.Len() {
    panic("Error: index out of bounds")
}
return slice.Index(i).Interface()
```

Comment (tkerambloch) [reply helpers/helpers.go]: ✔️

File: examples/helpers/example.txt.tmpl [hunk 2]
```diff
@@ -46,6 +46,9 @@
 {{`{{regexReplaceAllLiteral "a(x*)b" "-ab-axxb-" "${1}"}}`}}:                             {{regexReplaceAllLiteral "a(x*)b" "-ab-axxb-" "${1}"}}
 {{`{{regexSplit "z+" "pizza" -1}}`}}:                                                     {{regexSplit "z+" "pizza" -1}}
 
+# Get one specifique methode on array method using index
```
[the remark this answers is not in the document]
Comment (tkerambloch) [reply examples/helpers/example.txt.tmpl]: ✔️

File: examples/helpers/example.txt [hunk 3]
```diff
@@ -45,6 +45,9 @@
 {{regexReplaceAllLiteral "a(x*)b" "-ab-axxb-" "${1}"}}:                             -${1}-${1}-
 {{regexSplit "z+" "pizza" -1}}:                                                     [pi a]
 
+# Get one specifique methode on array method using index
```
[the remark this answers is not in the document]
Comment (tkerambloch) [reply examples/helpers/example.txt]: ✔️

File: examples/helpers/example.txt [hunk 4]
```diff
@@ -45,6 +45,9 @@
 {{regexReplaceAllLiteral "a(x*)b" "-ab-axxb-" "${1}"}}:                             -${1}-${1}-
 {{regexSplit "z+" "pizza" -1}}:                                                     [pi a]
 
+# Get one specifique method on array method using index
```
Comment (jhayotte) [inline examples/helpers/example.txt]: typo: specific

Comment (tkerambloch) [reply examples/helpers/example.txt]: ✔️

**Pull request merged**